get-sysinternals-du.js

```diff
@@ -7,6 +7,8 @@ if (process.platform !== 'win32') {
   process.exit(0)
 }
 
-https.get('https://download.sysinternals.com/files/DU.zip', function (res) {
+const duZipLocation = process.env.FAST_FOLDER_SIZE_DU_ZIP_LOCATION ?? 'https://download.sysinternals.com/files/DU.zip';
```

Comment:
as that operator is only available in node 14+

```suggestion
const duZipLocation = process.env.FAST_FOLDER_SIZE_DU_ZIP_LOCATION || 'https://download.sysinternals.com/files/DU.zip';
```